src/network/protocol/messages/cmpctblock.zig

```diff
@@ -0,0 +1,170 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+
+const Sha256 = std.crypto.hash.sha2.Sha256;
+const BlockHeader = @import("../../../types/BlockHeader.zig");
+const CompactSizeUint = @import("bitcoin-primitives").types.CompatSizeUint;
+
+pub const CmpctBlockMessage = struct {
+    header: BlockHeader,
+    nonce: u64,
+    short_ids: []u64,
+    prefilled_txns: []PrefilledTransaction,
+
+    const Self = @This();
+
+    pub const PrefilledTransaction = struct {
+        index: CompactSizeUint,
+        tx: []u8, // Serialized transaction
```

Comment:
use Transaction, it's a vector of transactions

Reply:
Okay boss will get these done asap